ProteoformSuiteGUI/ExperimentTheoreticalComparison.cs

```diff
@@ -398,6 +441,84 @@ private void ET_Peak_List_DirtyStateChanged(object sender, EventArgs e)
             }
             updateFiguresOfMerit();
         }
+
+        private static string ET_relations_list()
+        {
+            string tsv_header = "experimental_mass\texperimental_RT\ttheoretical_mass\ttheoretical_accession\tmodification\t\tlysinecount\tdelta_mass";
+            List<string> rows = new List<string>();
+                foreach (ProteoformRelation relation in Lollipop.et_relations)
+                {
+                    ExperimentalProteoform e = (ExperimentalProteoform)relation.connected_proteoforms[0];
+                    TheoreticalProteoform t = (TheoreticalProteoform)relation.connected_proteoforms[1];
+                    rows.Add(String.Join("\t", e.agg_mass, e.agg_rt,t.modified_mass, t.accession, t.ptm_descriptions, t.lysine_count, relation.delta_mass));
+                 }
+            string results_rows = String.Join(Environment.NewLine, rows.Select(r => r.ToString()));
+            return tsv_header + "\n" + results_rows;
+        }
+
+        private void cb_notch_search_CheckedChanged(object sender, EventArgs e)
+        {
+            Lollipop.notch_search_et = cb_notch_search.Checked;
+            tb_notch_masses.Enabled = cb_notch_search.Checked;
+            if (cb_notch_search.Checked) tb_notch_masses.Text = "Enter notches to search, separated by semi-colon.";
+        }
+
+        private void cb_view_ed_CheckedChanged(object sender, EventArgs e)
+        {
+            if (cb_view_ed.Checked)
+            {
+                DisplayUtility.GraphRelationsChart(ct_ET_Histogram, Lollipop.ed_relations["DecoyDatabase_0"], "relations");
+            }
+            else
+            {
+                DisplayUtility.GraphRelationsChart(ct_ET_Histogram, Lollipop.et_relations, "relations");
+
+            }
+        }
+
+        private void cb_Graph_lowerThreshold_CheckedChanged(object sender, EventArgs e)
```

Comment:
I was trying to take screen shots for my TBO ppt/paper without any lines. Normally they're fine 
![image](https://cloud.githubusercontent.com/assets/16846041/24581815/40f69ac8-16e8-11e7-9d4d-bc2dad8ecd77.png)